spec/lib/import_users_spec.rb

```diff
@@ -25,12 +25,12 @@
   end
 
   it 'creates users from a csv file' do
-    headers = [:row_number, :username, :password_digest, :title, :first_name, :last_name, :full_name, :email_address]
```

Comment:
We might need to use the suffix field if we need to import users from Connexions again if we're getting rid of `full_name`.

@reedstrm Are we still going to import users from Connexions?